### PR TITLE
Compile PDCurses if targeting windows

### PR DESCRIPTION
**Pull request by ppebb**
Adds MSBuild task to compile PDCurses if make (or mingw32-make on linux) is available and it is targeting windows.

Addresses task #79 and story #74

### CODE REVIEW

**Comment by ppebb** [comment]
Not sure whether compilation works on Windows, so I'd like someone to make sure it works.

I'd also appreciate feedback on improving the quality of the MSBuild code because it's ugly but I'm not sure how to improve it.

File: src/Blackguard/Blackguard.csproj [hunk 1]
```diff
@@ -38,6 +42,23 @@
...
+    <Exec Command="$(MakeBinary) -h > NUL" Condition="$(IsWindows)" IgnoreExitCode="True">
+      <Output TaskParameter="ExitCode" PropertyName="MakeExitCode" />
+    </Exec>
+    <Exec Command="$(MakeBinary) --help > /dev/null" Condition="$(IsLinux)" IgnoreExitCode="True">
+      <Output TaskParameter="ExitCode" PropertyName="MakeExitCode" />
+    </Exec>
+  </Target>
+
+  <!-- Compile PDCurses if targeting windows -->
+  <Target Name="Compile PDCurses" BeforeTargets="BeforeBuild" Condition="$(BuildingWindows)" DependsOnTargets="CheckMakeBinaryCallable">
```
Comment (LandonLeo) [inline src/Blackguard/Blackguard.csproj]: This should run even if an os-flag is not set when compiling (such as building normally on windows).

**Review by LandonLeo — CHANGES_REQUESTED** [review]

File: src/Blackguard/Blackguard.csproj (hunk 1 above)
Comment (ppebb) [reply src/Blackguard/Blackguard.csproj]: Addressed in [b41992d](https://github.com/ppebb/isp2/pull/80/commits/b41992d2be11f1deb504803ea0f4d27e12cba77e)

**Review by LandonLeo — COMMENTED** [review]
This pull works on windows and provides no direct issues.

**Review by LandonLeo — APPROVED** [review]

**Pull request merged**